### PR TITLE
add `$this->Flash->render()` to default layout

### PR DESCRIPTION
**Pull request by ayman-alkom**
I didn't edit examples files, because I didnt use them.
But I think they need also

### CODE REVIEW

File: src/Template/Layout/default.ctp [hunk 1]
```diff
@@ -85,6 +85,7 @@ $this->prepend('script', $this->Html->script(['jquery/jquery', 'bootstrap/bootst
 
     <?php
     echo $this->fetch('tb_body_start');
+    echo $this->Flash->render();
```
Comment (jadb) [inline src/Template/Layout/default.ctp]: Maybe better moved inside the `tb_body_start` on line 39?

**Comment by ayman-alkom** [comment]
What about the examples templates, does they need Flash statement?
I saw that it is very important for the default, but for others I don't know because I haven't used them yet.

**Comment by jadb** [comment]
@ayman-alkom they definitely could use it. The more I think about it and the more I think that the call to Flash should be included within an IF statement (check if helper is loaded in the first place) and yes, in default and other sample templates.

**Comment by ayman-alkom** [comment]
Hi @jadb 
I have tested the code, let'us say that the `tb_body_start` block is initiated in the actions views, so I think that it is more likely to be outside the `tb_body_start` block.
CakePHP conventions generally think by this way.

Otherwise we have to edit the bake templates too.

**Comment by jadb** [comment]
Ok for `tb_flash`.

**Comment by ayman-alkom** [comment]
It is working well for now, I have tested it in my project.

File: src/Template/Layout/default.ctp [hunk 2]
```diff
@@ -38,6 +38,12 @@ if (!$this->fetch('tb_body_start')) {
     echo '<body' . $this->fetch('tb_body_attrs') . '>';
     $this->end();
 }
+if (!$this->fetch('tb_flash')) {
```
Comment (jadb) [inline src/Template/Layout/default.ctp]: New line and small docblock like for the others please.

**Comment by jadb** [comment]
Thanks! Can you modify the samples as well?

**Comment by jadb** [comment]
Please add dockblock as commented above (to samples as well). Only missing that before I can merge.

**Comment by ayman-alkom** [comment]
If you want the dockblocks for samples can be shorted as following `\\ .....`

**Comment by jadb** [comment]
It's good. Would have preferred new lines to separate stuff, but it's good. I am merging.

**Pull request merged**